static/sass/_snapcraft_snap-details.scss

```diff
@@ -0,0 +1,11 @@
+@mixin snapcraft-snap-details {
+  @media screen and (max-width: $breakpoint-small) {
+    .snapcraft-snap-details {
+      &__snippet {
+        left: -4.5rem;
```

Comment:
If you do it with `margin-left: -4.5rem` it adapts width automatically without whole calc magic.
Still, a hack that I don't like too much.

Would be better to move install part out of media object and move to separate grid row with a spacing col on desktop.
But this will probably not line up well with the icon above, so...

Hack is faster to implement, so probably good enough for now, and we probably should look into redoing this part of page (when there will be channel maps, badges, etc - it will get more complicated).

Reply:
Done!